### PR TITLE
Change exposure generator

### PR DESCRIPTION
**Pull request by crisingulani**

### CODE REVIEW

File: backend/run.sh [hunk 1]
```diff
@@ -14,11 +14,13 @@ for package in desispec desiutil; do
...
 done
 
+export PYTHONPATH=$QLF_ROOT/framework/bin:$PYTHONPATH
+
 cd $QLF_ROOT
 echo "Initializing QLF Daemon..."
 
 # Start QLF daemon
-python -Wi framework/bin/qlf_daemon.py &> $QLF_ROOT/qlf_daemon.log &
+python -Wi framework/bin/servers.py &
```
Comment (felipelm) [inline backend/run.sh]: Are we not going to log the daemon anymore?

File: backend/framework/bin/exposure_generator.new.py [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+        self.min_interval = 2
+        self.max_interval = 5
+
+        cfg = configparser.ConfigParser()
+
+        try:
+            cfg.read(qlf_conf)
+
+            self.spectro_path = os.path.normpath(
+                cfg.get("namespace", "desi_spectro_data"))
```
[the remark this answers is not in the document]
Comment (crisingulani) [reply backend/framework/bin/exposure_generator.new.py]: This file was by mistake. :)

File: backend/run.sh (hunk 1 above)
Comment (crisingulani) [reply backend/run.sh]: I'm going to edit it. It's only a test.

File: backend/framework/bin/qlf_pipeline.py [hunk 3]
```diff
@@ -289,14 +277,45 @@ def resume_log(self, line, camera, lock):
...
+        'night': '20190101',
+        # 'cameras': [{'name': 'r7'}, {'name': 'r8'}],
+        'cameras': [{'name': 'r7'}],
+        'teldec': 14.84,
+        'telra': 333.22,
+        'exptime': 1000,
+        'zfill': '00000003',
+        'dateobs': '2019-01-01T22:00:00',
+        'tile': 6,
+        'desi_spectro_data': '/home/singulani/Projects/quicklook/qlf/backend/spectro/data',
```
[the remark this answers is not in the document]
Comment (crisingulani) [reply backend/framework/bin/qlf_pipeline.py]: I'm going to edit it. It's only a test.

File: backend/framework/bin/exposure_generator.py [hunk 4]
```diff
@@ -0,0 +1,251 @@
+import astropy.io.fits
```
[the remark this answers is not in the document]
Comment (crisingulani) [reply backend/framework/bin/exposure_generator.py]: `backend/framework/bin/exposure_generator.new.py` has been removed

File: backend/framework/bin/dos_monitor.py [hunk 5]
```diff
@@ -120,14 +120,14 @@ def get_exposure_info(self, filepath, night):
...
+        # dateobs = "%s-%s-%s 22:00" % (night[:-4], night[-4:][:2], night[-2:])
 
         return {
             'telra': hdr.get('telra', None),
             'teldec': hdr.get('teldec', None),
             'tile': hdr.get('tileid', None),
-            #'dateobs': hdr.get('date-obs', None),
-            'dateobs': dateobs,
+            'dateobs': hdr.get('date-obs', None),
+            # 'dateobs': dateobs,
```
Comment (felipelm) [inline backend/framework/bin/dos_monitor.py]: Same here as above

File: backend/framework/bin/qlf_pipeline.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (felipelm) [reply backend/framework/bin/qlf_pipeline.py]: Let's at least leave a `/full/path/to/spectro/data` placeholder

File: backend/framework/bin/qlf_pipeline.py [hunk 6]
```diff
@@ -291,14 +279,45 @@ def resume_log(self, line, camera, lock):
...
+        'desi_spectro_redux': '/home/singulani/Projects/quicklook/qlf/backend/spectro/redux',
```
Comment (felipelm) [inline backend/framework/bin/qlf_pipeline.py]: Here as well

**Pull request merged**